ateam_kenobi/src/play_selector.cpp

```diff
@@ -30,27 +30,29 @@
 namespace ateam_kenobi
 {
 
-PlaySelector::PlaySelector()
+PlaySelector::PlaySelector(rclcpp::Node & node)
```

Comment:
Would it make more sense to just pass the logger in here?

Reply:
I think I'll need the node in the next wave of work to add a parameter management system.